types/patches/0051-tsflower-rnsac-Fix-React.JSX.Element-s-that-didn-t-g.patch

```diff
@@ -0,0 +1,63 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Chris Bobbe <cbobbe@zulip.com>
+Date: Wed, 24 Jan 2024 13:50:36 -0800
+Subject: [tsflower] rnsac: Fix `React.JSX.Element`s that didn't get rewritten
+
+I think this is supposed to be taken care of by TsFlower...see
+prepGlobalJsxRewrites in rewrite/react.ts. Anyway, just do it
+manually, using the type that it looks like TsFlower should have
+used (`JSX$Element` in subst/react.js.flow).
```

Comment:
Hmm fun.

I agree this looks like the right answer. Rereading that code, what `prepGlobalJsxRewrites` says is that `JSX.Element` (with the global `JSX` namespace, not imported from anywhere) should get rewritten to that. So it seems likely that `React.JSX.Element` is intended to mean the same thing as `JSX.Element`.

So that function is a good place to look for a hint about what this type from React probably means. But I think the fix in principle is just another item in the list of rewrites.

Reply:
Hmm, OK. I've opened up the TsFlower code and I plan to make a PR doing that.